openstack/nova/templates/console-shellinabox-deployment.yaml

```diff
@@ -37,7 +38,8 @@ spec:
         runAsGroup: 33
         fsGroup: 33
 {{ tuple . "nova" "console-cell1-shellinabox" | include "kubernetes_pod_anti_affinity" | indent 6 }}
-{{ include "utils.proxysql.pod_settings" . | indent 6 }}
+      {{- include "utils.proxysql.pod_settings" . | nindent 6 }}
+      {{- tuple . (dict "name" "nova-console-shellinabox") | include "utils.topology.constraints" | indent 6 }}
```

Comment:
I think we need to rename this to `nova-console-cell1-shellinabox`.

Reply:
done